source/_docs/drupal-s3.md

```diff
@@ -0,0 +1,205 @@
+  //or if you have a contrib subfolder for modules use:
+  //terminus drush 'make --no-core ~/code/sites/all/modules/contrib/s3fs/s3fs.make ~/code/'</code></pre>
+  The above command will add the AWS SDK version 2.x library into the <code>sites/all/libraries/awssdk2</code> directory.
+ </div>
+  <div role="tabpanel" class="tab-pane" id="d8s3fs">
+   <br/>
+   <p>Install the <a href="https://www.drupal.org/project/s3fs">S3 File System</a> module and AWS SDK version 3.x library using Composer.</p>   
+   First, ensure that the Composer for your site will first look to Drupal's preferred package source to find modules:
+   <pre><code class="bash hljs">composer config repositories.drupal composer https://packages.drupal.org/8</code></pre>
+   Install s3fs module from the preferred package source:
+   <pre><code class="bash hljs">composer require drupal/s3fs --prefer-dist</code></pre>
+  </div>
+ </div>
+
+
+####S3 File System CORS
+Use the [S3 File System CORS Upload](https://www.drupal.org/project/s3fs_cors) module to enhance your Drupal media handling amd interface with your S3 bucket by having you file uploads go directly to you S3 bucket.
+ <!-- Nav tabs -->
+ <ul class="nav nav-tabs" role="tablist">
+  <li role="presentation" class="active"><a href="#d7s3fscors" aria-controls="d7s3fscors" role="tab" data-toggle="tab">Drupal 7</a></li>
+  <li role="presentation"><a href="#d8s3fscors" aria-controls="d8s3fscors" role="tab" data-toggle="tab">Drupal 8</a></li>
+ </ul>
+ <!-- Tab panes -->
+ <div class="tab-content">
+  <div role="tabpanel" class="tab-pane active" id="d7s3fscors">
+  <br/>
+  Install s3fs_cors module using Drush:
+  <pre><code class="bash hljs">terminus drush 'en jquery_update s3fs_cors -y'</code></pre>
+ </div>
+  <div role="tabpanel" class="tab-pane" id="d8s3fscors">
+   <br/>
+   Install s3fs_cors module using Composer:
+   <pre><code class="bash hljs">composer require drupal/s3fs_cors --prefer-dist</code></pre>
+  </div>
+ </div>
+
+#### Media Modules (Optional)
+
+You may want to enhance your Drupal media handling in conjunction with using AWS S3.
+<!-- Nav tabs -->
+<ul class="nav nav-tabs" role="tablist">
+ <li role="presentation" class="active"><a href="#d7media" aria-controls="d7media" role="tab" data-toggle="tab">Drupal 7</a></li>
+ <li role="presentation"><a href="#d8media" aria-controls="d8media" role="tab" data-toggle="tab">Drupal 8</a></li>
+</ul>
+<!-- Tab panes -->
+<div class="tab-content">
+ <div role="tabpanel" class="tab-pane active" id="d7media">
+ <br/>
+ To have solid, viewable, manageable Drupal 7 file handling (See: <a href="https://www.drupal.org/node/1699054">Media 2.x Quick Start Guide</a>), install:<br />   
+ Ctools, File Entity, Views, and Entity View Modes
+ <pre><code class="bash hljs">terminus drush 'en ctools file_entity views entity_view_mode -y'</code></pre>
+
+ Then install the Media module 2.x (e.g. 7.x-2.x-dev)
```

Comment:
Yes. It is. Missed that. This whole section on installing Media module is actually extraneous. It is my own prior  experience with Media module that leads me to suggest this. This should be removed to keep this as clean and simple as possible.